tests/test_benchmark/test_benchmark_integration_with_sentencetransformers.py

```diff
@@ -0,0 +1,31 @@
+"""test mteb.MTEB's integration with SentenceTransformers"""
+
+from __future__ import annotations
+
+import logging
+from typing import Union
+
+import pytest
+from sentence_transformers import SentenceTransformer
+
+from mteb import MTEB
+from mteb.abstasks import AbsTask
+
+from .task_grid import MOCK_TASK_TEST_GRID
+
+logging.basicConfig(level=logging.INFO)
+
+
+@pytest.mark.parametrize("task", MOCK_TASK_TEST_GRID)
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        "average_word_embeddings_levy_dependency",
+    ],
+)
+def test_benchmark_sentence_transformer(task: Union[str, AbsTask], model_name: str):
+    """Test that a task can be fetched and run"""
+    if isinstance(model_name, str):
+        model = SentenceTransformer(model_name)
+    eval = MTEB(tasks=[task])
```

Comment:
Same here. Since this test is under `test_benchmark`, should it test a `Benchmark` object?

Reply:
Ideally, I would actually love to make the MTEB object into a Benchmark object. 


```
mybench = mteb.Benchmark(tasks = tasks)
mybench.run(,model)

# for an existing benchmark we could then do:
en_bench = mteb.get_benchmark("mteb(eng)")
en_bench.run(...)
```

This seems like a v2 change though.